plugins/plugin-chart-table/src/buildQuery.ts

```diff
@@ -46,6 +46,12 @@ export default function buildQuery(formData: TableChartFormData) {
   const { percent_metrics: percentMetrics, order_desc: orderDesc = false } = formData;
   const queryMode = getQueryMode(formData);
   const sortByMetric = ensureIsArray(formData.timeseries_limit_metric)[0];
+
+  // never include time in raw records mode
+  if (queryMode === QueryMode.raw) {
+    formData.include_time = false;
+  }
```

Comment:
I'm just disabling selecting the Time column for raw records mode. We may want to allow that in the future though because sometimes it's useful to add truncated dates even for raw records and the time column provides an easy way to do that.